features/order/managing_orders/order_details/seeing_order_shipping_taxes.feature

```diff
@@ -20,6 +20,7 @@ Feature: Seeing taxes of an order
     Scenario: Seeing taxes of order items and shipping
         Given the customer bought a single "Symfony2 T-Shirt"
         And the customer chose "DHL" shipping method to "United States" with "Offline" payment
+        And the order "#00000666" is already paid
```

Comment:
As far as I see, the scenario is working even without this step :D Can we think about any scenario that does not work with the proposed fix but works with it?

Reply:
I'm also not fully convinced to this approach, but I've added a second scenario to check proper taxes on paid order